lib/rex/proto/drda/utils.rb

```diff
@@ -111,7 +111,7 @@ def self._info_secchkrm(ddm)
         next
       end
     end
-    if info_hash[:serverity].to_i.zero? and info_hash[:security_check_code].to_i.zero?
+    if info_hash[:severity].to_i.zero? and info_hash[:security_check_code].to_i.zero?
```

Comment:
Someone should manually review this, I didn't see when the hash was created any reference to `severity` or `serverity`. My guess is its been broken forever, but should def be reviewed.

Reply:
It should be `severity_code` based on the assignment above https://github.com/rapid7/metasploit-framework/blob/ee3b8b40b690f6209cd3d8797eb26799e6e07565/lib/rex/proto/drda/utils.rb#L107

Reply:
```suggestion
    if info_hash[:severity_code].to_i.zero? and info_hash[:security_check_code].to_i.zero?
```